uber/templates/signups/shifts.html

```diff
@@ -193,27 +187,18 @@
         } else {
             confirmStr += "signup ";
         }
-        confirmStr += title + " at " + time + "?";
+        confirmStr += title + " at " + time + " on " + date + "?";
         var r = confirm(confirmStr);
         if (r == true) {
         	var tgtShiftURL = "sign_up?job_id="+ id;
             if(state=='true') {
                 tgtShiftURL = "drop?job_id=" + id;
             }
             $.post( tgtShiftURL,postData,function( data,status ) {
-                window.location.href = window.location.pathname + "?tgt_date=" + moment(event.start).toISOString();
+                var view = $('#shift_cal').fullCalendar('getView');
+                window.location.href = window.location.pathname + "?view=" + view.name + "&start=" + view.start.toISOString();
```

Comment:
We use the view.start instead of event.start because it's better at preserving the view (thus making adding/dropping shifts seamless). Otherwise someone in List view would have their list reset to a start date of whatever shift they just added or dropped.

Ideally we'd skip this stuff and Ajaxify this, but that's a refactor for another day.